libs/core/src/components/sage-button/stories/sage-button.stories.mdx

```diff
@@ -0,0 +1,136 @@
+
+<Canvas>
+  <Story name="Secondary" args={{ variant: 'secondary', icon: '', disabled: false, slot: 'Default', type: 'button' }}>
+    {Template.bind()}
+  </Story>
+  <Story name="SecondaryDisabled" args={{ variant: 'secondary', icon: '', disabled: true, slot: 'Disabled', type: 'button' }}>
+    {Template.bind()}
+  </Story>
+  <Story name="SecondaryIcon" args={{ variant: 'secondary', icon: 'trashIcon', disabled: false, slot: 'Icon', type: 'button' }}>
+    {Template.bind()}
+  </Story>
+</Canvas>
+
+### Accent
+
+Accent button styles, used for tertiary actions or when more contrast is needed
+
+<Canvas>
+  {/* <div> */}
+  <Story name="Accent" args={{ variant: 'accent', icon: '', disabled: false, slot: 'Default', type: 'button' }}>
+    {Template.bind()}
+  </Story>
+  <Story name="AccentDisabled" args={{ variant: 'accent', icon: '', disabled: true, slot: 'Disabled', type: 'button' }}>
+    {Template.bind()}
+  </Story>
+  <Story name="AccentIcon" args={{ variant: 'accent', icon: 'trashIcon', disabled: false, slot: 'Icon', type: 'button' }}>
+    {Template.bind()}
+  </Story>
+  {/* </div> */}
+</Canvas>
+
+### Disclosure
+
+Disclosure button styles, used to provide multiple options or more information within a button
+
+<Canvas>
+  <Story name="Disclosure" args={{ variant: 'disclosure', icon: '', disabled: false, slot: 'Default', type: 'button' }}>
+    {Template.bind()}
+  </Story>
+  <Story name="DisclosureDisabled" args={{ variant: 'disclosure', icon: '', disabled: true, slot: 'Disabled', type: 'button' }}>
+    {Template.bind()}
+  </Story>
+  <Story name="DisclosureIcon" args={{ variant: 'disclosure', icon: 'trashIcon', disabled: false, slot: 'Icon', type: 'button' }}>
+    {Template.bind()}
+  </Story>
+</Canvas>
+
+### Destructive
+
+Destructive button styles, used with actions that delete information such as deleting an item or removing a contact
+
+<Canvas>
+  <Story name="Destructive" args={{ variant: 'destructive', icon: '', disabled: false, slot: 'Default', type: 'button' }}>
+    {Template.bind()}
+  </Story>
+  <Story name="DestructiveDisabled" args={{ variant: 'destructive', icon: '', disabled: true, slot: 'Disabled', type: 'button' }}>
+    {Template.bind()}
+  </Story>
+  <Story name="DestructiveIcon" args={{ variant: 'destructive', icon: 'trashIcon', disabled: false, slot: 'Icon', type: 'button' }}>
+    {Template.bind()}
+  </Story>
+</Canvas>
+
+## Type
+
+The type prop allows a `sage-button` to set [button type attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type).
+
+### Default (Button)
+
+The button has no default behavior, and does nothing when pressed by default. It can have client-side scripts listen to the element's events, which are triggered when the events occur.
+
+This is our default button type.
+
+### Submit
+
+The button submits the form data to the server.
+
+### Reset
+
+The button resets all the controls to their initial values. (This behavior tends to annoy users.)
```

Comment:
`(This behavior tends to annoy users.)`
Do you have an accessibility article or ux documentation to support this claim? I don't think this portion of documentation should be opinionated.

Reply:
I'll push a commit with a link to MDN but since MDN calls it out, I figured its valuable for us to note it in our docs as well. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type

Reply:
If they call it out, I suppose it's fine. I'm confused by that wording from them because the intent of the `reset` property is to reset all the form fields. Not sure how that would annoy users unless it's not being used correctly. But alas,